pkg/karmadactl/top/top_pods.go

```diff
@@ -115,13 +115,16 @@ func NewCmdTopPod(f util.Factory, parentCommand string, o *PodOptions, streams g
 		Short:                 i18n.T("Display resource (CPU/memory) usage of pods of member clusters"),
 		Long:                  topPodLong,
 		Example:               fmt.Sprintf(topPodExample, parentCommand),
-		ValidArgsFunction:     completion.ResourceNameCompletionFunc(f, "pod"),
+		ValidArgsFunction:     utilcomp.ResourceNameCompletionFunc(f, "pod"),
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, cmd, args))
 			cmdutil.CheckErr(o.Validate())
 			cmdutil.CheckErr(o.RunTopPod(f))
 		},
 		Aliases: []string{"pods", "po"},
+		Annotations: map[string]string{
+			"parent": "top",
```

Comment:
```suggestion
			"parent": "top", // used for completion code to set default operation scope.
```
I suggest adding a comment for this annotation.